xla/debug_options_flags.cc

```diff
@@ -81,7 +81,7 @@ DebugOptions DefaultDebugOptionsIgnoringFlags() {
 
   // TODO(b/258036887): Enable cuda_graph_level=2. Currently blocked by CUDA 12
   // integration.
-  opts.set_xla_gpu_cuda_graph_level(1);
+  opts.set_xla_gpu_cuda_graph_level(0);
```

Comment:
I guess this was not an intentional change (probably merge conflict?). Please revert the change in this file.

Reply:
Yes, I think this is rollback to 0 now. https://github.com/openxla/xla/pull/3959